images/ansible/filter_plugins/kube.py

```diff
@@ -1,3 +1,17 @@
+# Copyright 2018 Heptio Inc.
```

Comment:
s/Heptio/The Kubernetes Authors/

We should probably have this on all the Ansible yaml files as well.